strings/traitor_flavor.json

```diff
@@ -0,0 +1,106 @@
+		"uplink_name": "syndicate uplink",
+		"uplink_theme": "syndicate"
+	},
+	"Donk Corporation": {
+		"introduction": "You are the Donk Co. Traitor.",
+		"allies": "Members of Waffle Co. are to be killed on sight; they are not allowed to be on the station while we're around.",
+		"goal": "We do not approve of mindless killing of innocent workers; \"get in, get done, get out\" is our motto.",
+		"uplink": "You have been provided with a standard uplink to accomplish your task.",
+		"uplink_name": "syndicate uplink",
+		"uplink_theme": "syndicate"
+	},
+	"Waffle Corporation": {
+		"introduction": "You are the Waffle Co. Traitor.",
+		"allies": "Members of Donk Co. are to be killed on sight; they are not allowed to be on the station while we're around. Do not trust fellow members of the Waffle.co (but try not to rat them out), as they might have been assigned opposing objectives.",
+		"goal": "You are not here for a stationwide demonstration. Again, other Waffle Co. Traitors may be, so watch out. Your job is to only accomplish your objectives.",
+		"uplink": "You have been provided with a standard uplink to accomplish your task.",
+		"uplink_name": "syndicate uplink",
+		"uplink_theme": "syndicate"
+	},
+	"Gone Postal": {
+		"introduction": "You are the Tiger Cooperative Fanatic.",
+		"allies": "If the syndicate learns of your plan, they're going to kill you and take your uplink. Take no chances.",
+		"goal": "The preparations are finally complete. Today is the day you go postal. You're going to hijack the emergency shuttle and live a new life free of Nanotrasen.",
+		"uplink": "You've actually managed to steal a full uplink a month ago. This should certainly help accomplish your goals.",
+		"uplink_name": "syndicate uplink",
+		"uplink_theme": "syndicate"
+	},
+	"Internal Affairs Agent": {
+		"introduction": "You are the Internal Affairs Agent.",
+		"allies": "Death to the Syndicate. Killing infiltrators is a whole different kind of internal affair we fully approve of dealing with.",
+		"goal": "While you have a license to kill, unneeded property damage or loss of employee life will lead to your contract being terminated.",
+		"uplink": "For the sake of plausible deniability, you have been equipped with an array of captured Syndicate weaponry available via uplink.",
+		"uplink_name": "syndicate uplink",
+		"uplink_theme": "syndicate"
+	},
+	"Corporate Climber": {
+		"introduction": "You are the Corporate Climber.",
+		"allies": "Death to the Syndicate.",
+		"goal": "Killing needlessly would make you some kind of traitor, or at least definitely seen as one. This is all just a means to an end.",
+		"uplink": "You have connections to the black market for the deeds. Knock off a few loose weights, and your climb will be so much smoother.",
+		"uplink_name": "underworld market",
+		"uplink_theme": "neutral"
+	},
+	"Legal Trouble": {
+		"introduction": "You are in legal trouble.",
+		"allies": "Death to the Syndicate.",
```

Comment:
this is effectively writing friendly-traitor into the code, which is objectively bad gameplay we want to avoid